users/migrations/0002_useraccount_display_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.7 on 2020-08-03 23:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0001_initial'),
```

Comment:
is the initial `users` migration missing? I don't see it in this PR

Reply:
found my users 0001_initial in an old branch! don't know why I lost it :grin: